projects/packages/sync/src/class-defaults.php

```diff
@@ -290,6 +290,7 @@ public static function get_constants_whitelist() {
 	 * @var array Default whitelist of callables.
 	 */
 	public static $default_callable_whitelist = array(
+		'get_loaded_extensions'            => 'get_loaded_extensions',
```

Comment:
Do hosting providers ever disable this, akin to`phpinfo`. I haven't heard of that personally, but wanted to ask outloud.

Reply:
Yes, that's a good point. We should create a wrapper around it, thanks for the suggestion!